### PR TITLE
Reverse x, y, z axes

### PR DESCRIPTION
**Pull request by hwchen**
In axes_common.rs:

- in struct `AxisData`, add field `reverse: bool`.
- in `AxisData::new()`, initialize struct with `reverse: false`.
- in `AxisData::write_out_commands()`, write `reverse`.
- add `AxisData::set_reverse()` method.
- in `trait AxesCommon : AxesCommonPrivate` add `set_x_reverse()` and
  `set_y_reverse()` methods.

In axes3d.rs:

- add method `set_z_reverse` to `Axes3D`.

### CODE REVIEW

**Comment by hwchen** [comment]
Hi! Tried to make it consistent with the existing code and your API suggestion in issue #16.

File: src/axes_common.rs [hunk 1]
```diff
@@ -319,7 +321,13 @@ impl AxisData
 			Fix(v) => write!(w, "{:.12e}", v),
 			Auto => w.write_str("*")
 		};
-		w.write_str("]\n");
+        if self.reverse
```
Comment (SiegeLord) [inline src/axes_common.rs]: These lines need tabs instead of spaces.

**Comment by hwchen** [comment]
I'll make the changes to accept a bool. Funny, that was actually how I implemented it first!

**Comment by hwchen** [comment]
Made the changes. I named the parameter `reverse`, let me know if you'd prefer another name.

**Comment by SiegeLord** [comment]
That looks good. Thanks a lot!

**Comment by hwchen** [comment]
Thank you!

**Pull request merged**